src/cookbooks/bigquery/access.md

```diff
@@ -130,19 +149,26 @@ gcloud beta dataproc clusters create cluster-name \
     --project project-id
 ```
 
-Jupyter URL can be retrieved with the following command:
+The Jupyter URL can be retrieved with the following command:
+
 ```bash
 gcloud beta dataproc clusters describe cluster-name --region=us-west1 --project project-id | grep Jupyter
 ```
 
 After you've finished your work, it's a good practice to delete your cluster:
+
 ```bash
 gcloud beta dataproc clusters delete cluster-name --region=us-west1 --project project-id --quiet
 ```
 
-## From Colaboratory
-[Colaboratory](https://colab.research.google.com) is Jupyter notebook environment, managed by Google and running in the cloud. Notebooks are stored in Google Drive and can be shared in a similar way to Google Docs.
+### Colaboratory
+
+> This method requires [BigQuery Access](#BigQuery-Access-Request) to be provisioned..
+
+[Colaboratory](https://colab.research.google.com) (Colab) is Jupyter notebook environment, managed by Google and running in the cloud. Notebooks are stored in Google Drive and can be shared in a similar way to Google Docs.
+
+Colab can be used to easily access BigQuery and perform analyses. See the [`Telemetry Hello World` notebook](https://colab.research.google.com/drive/1uXmrPnqzDATiCVH2RNJKD8obIZuofFHx) for an interactive example. Under the hood, it uses the BigQuery API to read and write to BigQuery tables, so access needs to be explicitly provisioned.
```

Comment:
```suggestion
You can use Colab to get access to BigQuery easily and perform analyses. See the [`Telemetry Hello World` notebook](https://colab.research.google.com/drive/1uXmrPnqzDATiCVH2RNJKD8obIZuofFHx) for an interactive example. It uses the BigQuery API to read and write to BigQuery tables. Therefore, access needs to be provisioned explicitly.
```

Reply:
I think some meaning got lost here-- same problem with the "therefore" and "access" as above. I think I'm going to leave this one as-is.